web/src/test/java/se/inera/intyg/rehabstod/auth/RehabstodUserDetailsServiceTest.java

```diff
@@ -373,7 +375,7 @@ public void testRemoveEnheterMissingRehabKoordinatorRole() {
 
         // Test
         userDetailsService.removeEnheterMissingRehabKoordinatorRole(original,
-            systemRoles.stream().map(rt -> rt.getRole()).collect(Collectors.toList()),
```

Comment:
I know that "rt -> rt.getRole()" vs  "HsaSystemRole::getRole" scenarios get automatic comments. Me myselt think the first one is more readable but that is just a matter in my opinion. :)